plugin/mouse/mouse_scroll.py

```diff
@@ -184,7 +195,13 @@ def mouse_scroll_continuous(new_scroll_dir: Literal[-1, 1]):
 
 def scroll_continuous_helper():
     scroll_amount = settings.get("user.mouse_continuous_scroll_amount")
-    y = scroll_amount * scroll_dir / 10
+    acceleration_setting = settings.get("user.mouse_continuous_scroll_acceleration")
+    acceleration_speed = (
+        1 + min((time.perf_counter() - scroll_ts) / 0.5, acceleration_setting - 1)
+        if acceleration_setting > 1
+        else 1
+    )
+    y = scroll_amount * acceleration_speed * scroll_dir / 10
```

Comment:
I know the existing code does this, but do we know why we have a setting for `mouse_continuous_scroll_amount` if we're just going to divide it by 10 arbitrarily 😭. Maybe the setting used to be different and we needed to change its meaning?

Reply:
I have no idea why and was very tempted to change it, but that would of course break every one who has changed the default setting.

Reply:
One of many great mysteries lost to time. Might be worth writing up an issue if either if you feel strongly that we should revisit

Reply:
Agreed
https://github.com/talonhub/community/issues/1605